test.cpp

```diff
@@ -845,7 +845,19 @@ tests["autograd/~integration/cartpole"] = []() {
 
 };
 
- return tests;
+tests["autograd/random/seed"] = []() {
```

Comment:
i wonder if that is sufficient?

Reply:
A GPU test would be nice... Although I have a feeling it might not work reliably.